libafl_sugar/src/inprocess.rs

```diff
@@ -158,24 +155,43 @@ where
             let edges_observer = HitcountsMapObserver::new(unsafe {
                 StdMapObserver::from_mut_slice(
                     "edges",
-                    OwnedMutSlice::from_raw_parts_mut(edges_map_mut_ptr(), self.map_size),
+                    OwnedMutSlice::from_raw_parts_mut(
+                        edges_map_mut_ptr(),
+                        EDGES_MAP_ALLOCATED_SIZE,
+                    ),
                 )
             })
             .track_indices();
 
             let cmplog_observer = CmpLogObserver::new("cmplog", true);
 
+            // New maximization map feedback linked to the edges observer and the feedback state
+            let map_feedback = MaxMapFeedback::with_name("map_feedback", &edges_observer);
+            // Extra MapFeedback to deduplicate finds according to the cov map
+            let map_objective = MaxMapFeedback::with_name("map_objective", &edges_observer);
+
+            let calibration = CalibrationStage::new(&map_feedback);
+
+            let stats_stage = AflStatsStage::builder()
```

Comment:
why stat?

Reply:
No reason not to have stats right

Reply:
but stage won't show you stat. it'll just send a json to the manager. 
also i think anything beyond corpus, execs, stability, .. etc is useless.  and for those you don't need aflstatsstage

Reply:
It does put some stuff in the TUI at least, and doesn't cost much, I always add it :D 
Maybe a placebo?

Reply:
if you use tui yes, because tui actually parse it. but here you are using MultiMonitor